LvivCompany.Bookstore/LvivCompany.Bookstore.Web/Views/Home/Index.cshtml

```diff
@@ -1,102 +1,35 @@
-                    <span class="glyphicon glyphicon-star"></span>
-                    <span class="glyphicon glyphicon-star"></span>
-                    <span class="glyphicon glyphicon-star"></span>
-                    <span class="glyphicon glyphicon-star"></span>
-                    <span class="glyphicon glyphicon-star-empty"></span>
-                </div>
-                <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. </p>
-                <hr class="line">
-                <div class="row">
-                    <div class="col-md-6 col-sm-6">
-                        <p class="price">$29,90</p>
-                    </div>
-                    <div class="col-md-6 col-sm-6">
-                        <button class="btn btn-success right"> BUY ITEM</button>
-                    </div>
+@model List<BookDetailViewModel>
 
-                </div>
-            </span>
-        </div>
-        <div class="col-md-3 col-sm-6">
-            <span class="thumbnail">
-                <img src="http://placehold.it/500x400" alt="...">
-                <h4>Product Tittle</h4>
-                <div class="ratings">
-                    <span class="glyphicon glyphicon-star"></span>
-                    <span class="glyphicon glyphicon-star"></span>
-                    <span class="glyphicon glyphicon-star"></span>
-                    <span class="glyphicon glyphicon-star"></span>
-                    <span class="glyphicon glyphicon-star-empty"></span>
-                </div>
-                <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. </p>
-                <hr class="line">
-                <div class="row">
-                    <div class="col-md-6 col-sm-6">
-                        <p class="price">$29,90</p>
-                    </div>
-                    <div class="col-md-6 col-sm-6">
-                        <button class="btn btn-success right"> BUY ITEM</button>
-                    </div>
+ <form asp-controller="Home" asp-action="BookDetail">
+        <div class="container">
+            <div class="row">
+    <!-- BEGIN PRODUCTS -->
+
+    @foreach (var std in @Model)
+    {
 
-                </div>
-            </span>
-        </div>
         <div class="col-md-3 col-sm-6">
             <span class="thumbnail">
-                <img src="http://placehold.it/500x400" alt="...">
-                <h4>Product Tittle</h4>
-                <div class="ratings">
-                    <span class="glyphicon glyphicon-star"></span>
-                    <span class="glyphicon glyphicon-star"></span>
-                    <span class="glyphicon glyphicon-star"></span>
-                    <span class="glyphicon glyphicon-star"></span>
-                    <span class="glyphicon glyphicon-star-empty"></span>
-                </div>
-                <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. </p>
+                <img src="@std.Image" alt="...">
```

Comment:
Alt attribute should contain some meaningful text. It could be book name for example